source/includes/_processing.md

```diff
@@ -0,0 +1,314 @@
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+`name` <br> *string* | name of the processor
+`args` <br> *object* | processor-dependent parameters
+
+
+### Returns
+The newly created [processor object](#the-processor-object).
+
+
+## Retrieve informations about a processor
+
+This endpoint is meant to retrieve a processor object in a dataset processing stack from its uid.
+
+> Definition
+
+```HTTP
+GET https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XGET https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_newnew
+    -u username:password
+```
+
+> Example response
+
+```json
+{
+    "processor_uid": "pr_newnew",
+    "name": "string_extractor",
+    "args": {
+        "field": "text_field",
+        "regexp": "(?P<planet>^World)"
+    }
+}
+```
+
+### Returns
+The requested [processor object](#the-processor-object).
+
+## Update a processor
+
+This endpoint is meant to update a processor in place within the processing stack of a dataset.
+Explanation of the thing
+
+> Definition
+
+```
+PUT https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XPUT https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_abcdef
+    -u username:password \
+    -d '{"name":"regexp_replace", "args": {"field": "text_field", "regexp": ".*", "new": "Bonjour Monde"}'
+```
+
+> Example response
+
+```json
+{
+    "processor_uid": "pr_abcdef", 
+    "name":"regexp_replace",
+    "args": {
+        "field": "text_field",
+        "regexp": ".*",
+        "new": "Hello World"
+    }
+}
+```
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+`name` <br> *string* | name of the processor
+`args` <br> *object* | processor-dependent parameters
+
+### Returns
+The newly updated [processor object](#the-processor-object).
+
+## Delete a processor
+
+This endpoint is meant to delete a processor from the processing stack of a dataset
+
+> Definition
+
+```HTTP
+DELETE https://{YOURDOMAIN}.opendatasoft.com/api/management/v2/datasets/{DATASET_UID}/processors/{PROCESSOR_UID}
+```
+
+> Example request
+
+```HTTP
+curl -XDELETE https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_abcdef/processors/pr_newnew
+    -u username:password
+```
+
+### Returns
+
+On successful deletion, the endpoint returns a HTTP 204 without any content.
+
+
+## Retrieve possible processor parameters from processor name
+
+For some processors and some processors parameters, the OpenDataSoft platform has a way of determining a set of possible values. It can for instance figure out the list of fields the processor could operate on.
```

Comment:
I think it would be more clear to say something like "to suggest values for the processor parameters", because I didn't understand on first read.

Reply:
It sounds good but I don't know if it is accurate enough

Reply:
I think it's a good suggestion :+1: